test/models/user_test.rb

```diff
@@ -620,7 +618,7 @@
             create_list(:activity, 2, type_id: Activity.type_ids[activity_type], user_id: @user.id, created_at: @created_at)
           end
 
-          it "2であること" do
+          it '2であること' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.